client/src/partials/patient_invoice/registry/registry.js

```diff
@@ -0,0 +1,101 @@
+angular.module('bhima.controllers')
+.controller('InvoiceRegistryController', InvoiceRegistryController);
+
+InvoiceRegistryController.$inject = [
+  'PatientInvoiceService', '$uibModal', 'NotifyService', 'util', 'ReceiptModal'
+];
+
+/**
+ * Invoice Registry Controller
+ *
+ * This module is responsible for the management
+ * of Invoice Registry.
+ *
+ */
+function InvoiceRegistryController(Invoices, $uibModal, Notify, util, Receipt) {
+    var vm = this;
+
+    var invoiceActionsTemplate =
+        '<div style="padding : 5px"><a ng-click="grid.appScope.showBill(row.entity.uuid)">' +
```

Comment:
If you put `href` on this `<a>` tag, it will give a nice little hand icon to click on.